.github/workflows/test.yml

```diff
@@ -23,6 +25,9 @@ jobs:
         run: |
           yarn --ignore-engines
           yarn build
-          yarn test
+          # yarn test
+          npx jest test/dev/dev.test.ts
```

Comment:
Note: due to the nature of this test, I wouldn’t mind moving this to its own `dev` job where we don’t run it on Windows (maybe we only run it on 1 version of Node, too, to start).

I’m concerned about this test potentially taking a long time to run, and this is just my opinion but I think the build test is where we really need to test Windows & multiple Node versions; the dev server should behave predictably in many environments assuming the build is working as expected.

Reply:
yeah I agree. sorry for all the changes here in test.yml, it's because the tests get stuck, but only on CI, I made the changes for debugging only